src/client/Client.cc

```diff
@@ -8115,18 +8115,14 @@ loff_t Client::lseek(int fd, loff_t offset, int whence, const UserPerm& perms)
   if (f->flags & O_PATH)
     return -EBADF;
 #endif
-  return _lseek(f, offset, whence, perms);
+  return _lseek(f, offset, whence);
 }
 
-loff_t Client::_lseek(Fh *f, loff_t offset, int whence, const UserPerm& perms)
+loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
 {
   Inode *in = f->inode.get();
   int r;
 
-  if (!may_open(f->inode.get(), f->flags, perms)) {
-    return -EPERM;
-  }
```

Comment:
We may have high-debug client logs? But probably logging the warning there would be easier.